<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Forsyth Family Practice Center - MOA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/qrc/logo.qrc">
    <normaloff>:/newPrefix/imgs/logo.png</normaloff>:/newPrefix/imgs/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/imgs/app-bg.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="startWindow_Heading1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>12</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-image: none;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Medical Office</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="startWindow_Heading2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>28</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-image: none;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Administration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	border-image: none;
	background-color: none;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="assets/qrc/logo.qrc">:/newPrefix/imgs/logo.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="startWindow_SubHeading">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>18</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-image: none;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Forsyth Family Practice Center</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="startWindow_UsernameLineEdit">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>270</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border-image: none;
    border: none;
	min-width: 200px;
	height: 30px;
	background-color: #F3ECB0;
	color: #344D67;
	font-family: &quot;MS Shell Dlg 2&quot;;
	font-size: 11;
	padding-left: 10px;
	
}</string>
             </property>
             <property name="placeholderText">
              <string>Enter Username</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="startWindow_PasswordLineEdit">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>270</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	border-image: none;
    border: none;
	min-width: 200px;
	height: 30px;
	background-color: #F3ECB0;
	color: #344D67;
	font-family: &quot;MS Shell Dlg 2&quot;;
	font-size: 11;
	padding-left: 10px;
	
}</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Enter Password</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
           <item>
            <widget class="QLabel" name="ShowPassword_Label">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	border-image: none;
	border-radius: 5px;
	background-color: white;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="assets/qrc/show.qrc">:/newPrefix/imgs/show.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignBottom|Qt::AlignHCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="HidePassword_Label">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	border-image: none;
	border-radius: 5px;
	background-color: white;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="assets/qrc/hide.qrc">:/newPrefix/imgs/hide.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_3">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>120</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="loginErrorLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	border-image: none;
	background-color: none;
	color: white;
}</string>
         </property>
         <property name="text">
          <string>Login Error: Username/Password does not exsist</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="startWindow_LoginBtn">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-image: none;
	background-color: #6ECCAF;
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(139, 231, 100);
	color: white;
}</string>
           </property>
           <property name="text">
            <string>LOGIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startWindow_ExitBtn">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>90</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Lato</family>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-image: none;
	background-color: rgb(30, 147, 143);
	color: black;
	border: 2px solid black;
}

QPushButton::hover {
	background-color: rgb(14, 81, 85);
	color: white;
}</string>
           </property>
           <property name="text">
            <string>EXIT</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="styleSheet">
       <string notr="true">QFrame {
	border-image: none;
	background-color: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QPushButton" name="startWindow_InfoBtn">
         <property name="font">
          <font>
           <family>Lato</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	border-image: none;
	color: #F3ECB0;
	background-color: transparent;
	padding-bottom: 3px;
}

QPushButton::hover {
	color: rgb(241, 243, 103);
}</string>
         </property>
         <property name="text">
          <string>🛈</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/qrc/app-bg.qrc"/>
  <include location="assets/qrc/logo.qrc"/>
  <include location="assets/qrc/hide.qrc"/>
  <include location="assets/qrc/show.qrc"/>
  <include location="assets/qrc/doctor.qrc"/>
  <include location="../../../../../Users/27par/Desktop/Github Repos/Medical-Office-Administration/frontend/ui/assets/qrc/logo.qrc"/>
  <include location="../../../../../Users/27par/Desktop/Github Repos/Medical-Office-Administration/frontend/ui/assets/qrc/hide.qrc"/>
  <include location="../../../../../Users/27par/Desktop/Github Repos/Medical-Office-Administration/frontend/ui/assets/qrc/show.qrc"/>
 </resources>
 <connections/>
</ui>
